packages/edit-post/src/components/layout/style.native.scss

```diff
@@ -24,3 +24,15 @@
 	right: 0;
 	left: 0;
 }
+
+.offlineStatusContainer {
+	background-color: $light-ultra-dim;
+	padding: $grid-unit;
+	justify-content: center;
+	flex-direction: row;
+}
+
+.offlineText {
+	padding-left: 3;
+	padding-top: 2;
+}
```

Comment:
If we ultimately relocate these styles to its own component file, it may make sense to follow BEM selector naming as I believe that is our current project practice.

Reply:
I wasn't aware that BEM selector naming was the current practice. Looking at other native scss files, it appears to be [quite a mix](https://github.com/WordPress/gutenberg/blob/7b1f24c60e7ee71f3391349acf58d14f0bd03ce1/packages/block-library/src/image/styles.native.scss#L43-L70). I'm not against using BEM however.

Reply:
I'll share some context via a direct message.